twiggy/tests/expectations/garbage

```diff
@@ -0,0 +1,11 @@
+ Bytes │ Size % │ Item
+───────┼────────┼────────
+    11 ┊ 13.10% ┊ code[2]
+     8 ┊  9.52% ┊ code[1]
```

Comment:
We should probably also build `garbage.wasm` with the `--debug-names` flag, so we get function names instead of `code[N]`.